ext/opcache/ZendAccelerator.c

```diff
@@ -629,13 +615,70 @@ static void accel_use_shm_interned_strings(void)
 		auto_global = (zend_auto_global*)Z_PTR(p->val);;
 
 		zend_string_addref(auto_global->name);
-		auto_global->name = accel_new_interned_string(auto_global->name);
+		auto_global->name = new_interned_string(auto_global->name);
+		if (p->key) {
+			p->key = new_interned_string(p->key);
+		}
+	}
+
+	for (idx = 0; idx < module_registry.nNumUsed; idx++) {
```

Comment:
All of this code should be using ZEND_HASH_FOREACH_BUCKET... but not related to this change.